servers/rendering/renderer_rd/effects/copy_effects.h

```diff
@@ -155,6 +155,9 @@ class CopyEffects {
 		uint32_t pad2[2];
 		//SET color
 		float set_color[4];
+		// Depth.
+		bool inverse_depth;
+		uint32_t pad3[3];
```

Comment:
This should replace one of the pads above (I would replace ``pad``) There is no reason to introduce 3 new pads when there is already so much padding